reg-index/src/add.rs

```diff
@@ -41,6 +41,7 @@ pub fn add(
     index_url: &str,
     manifest_path: Option<&Path>,
     upload: Option<&str>,
+    force: bool,
```

Comment:
This will be a breaking API change.  Maybe a separate function `add_force` would be better? That would also make the api a little clearer.  Having random true/false values intermixed in a parameter list can be difficult to understand from the call site.

Reply:
Good point. I'll update the PR with this suggestion.

Additionally, are the version bumps that I did appropriate? Normally I wouldn't add something like this as a patch change, but wanted to try and follow a similar pattern to past changes.
I'm happy to make adjustments there too if necessary.

Reply:
The version bumps are fine.